wolox-books/src/app/screens/sign-up/mocks/mock-user.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from '../interfaces/user';
+export const MockUser: User = {
```

Comment:
Empty line between imports and another thinks.

Reply:
ok